### ISSUE TITLE
PSClickwrap Triggered Event callback props not working

### ISSUE THREAD

**Issue by arellin** [pactsafe/pactsafe-react-sdk#47]
**Setup:**

```
<PSClickWrap
    accessId={ACCESS_ID}
    groupKey={GROUP_KEY}
    clickWrapStyle='combined'
    signerId={signerId}
    onValid={(data) => onChange(true, data)}
    onInvalid={(data) => onChange(false, data)}
 />
```

**Steps to reproduce:**

1. Click agreement checkbox, onValid is triggered.
2. Uncheck agreement checkbox, onInvalid is not triggered.
3. Click agreement checkbox again, onValid is not triggered as well.

**Comment by emilykrebs** [comment]
Having this issue as well

**Comment by cdhcs1516** [comment]
@arellin @emilykrebs Thank you for posting this. The `onInvalid` callback function is bind with a Clickwrap group property `expect_disagreed`. So it won't be triggered when such property is not toggled on.

**Comment by AndriyPavliv** [comment]
Hi @cdhcs1516!
I'm using v2.5.1 and it is still reproduced. How can I set expect_disagreed to PSClickWrap component? I can't find appropriate prop in documentation.  Is there any documentation regarding this?

**Comment by cdhcs1516** [comment]
Hi @AndriyPavliv , if you are using our new React product, you should be able to set this allow_disagree behavior in editing the Clickwrap group.

**Comment by cdhcs1516** [comment]
@arellin @emilykrebs @AndriyPavliv Thanks for your feedbacks. The latest released v2.6.0 of `pactsafe-react-sdk` supports to set `allowDisagreed` attribute now to handle `onInvalid` events. Have a check on it.
